dragonchain/lib/broadcast.py

```diff
@@ -100,7 +119,9 @@ def send_receipts(l5_block: "l5_block_model.L5BlockModel") -> None:
                     claim_check_id = f"{chain_id}-{block}"
                     matchmaking.resolve_claim_check(claim_check_id)
                 except Exception:
-                    _log.exception("Failure to finalize claim in matchmaking. Sending reciepts to lower level nodes.")
+                    _log.exception("Failure to finalize claim in matchmaking. Sending receipts to lower level nodes.")
+                    _log.exception(f"Adding claim to failed queue.  Claim ID: {claim_check_id}")
```

Comment:
don't make 2 calls to the logger. Not only will this print a stack trace twice, it's unnecessary overhead calling the logger again when all you want is a newline

Reply:
Good point.  Fixed.